src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php

```diff
@@ -103,8 +103,8 @@ public function specifyShippingAddress(AddressInterface $shippingAddress): void
 
     public function selectShippingAddressProvince(string $province): void
     {
-        $this->waitForElement(5, 'shipping_country_province');
-        $this->getElement('shipping_country_province')->selectOption($province);
+        $this->waitForElement(5, 'shipping_country_province_code');
+        $this->getElement('shipping_province_code')->selectOption($province);
```

Comment:
Why it's waiting for a different element that action is performed on?

Reply:
I am not sure but, the country province is displaying by JS code, and here we waiting for loaded data 🗡